pkgs/development/python-modules/earcut/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  yq-go,
+  lib,
+  numpy,
+  fetchPypi,
+  runCommand,
+  setuptools,
+  buildPythonPackage,
+}:
+
+buildPythonPackage rec {
+  pname = "earcut";
+  version = "1.1.5";
+  pyproject = true;
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-IcVbAy/guQZY5/MF4e5AjLT3Z4LFBm2GJzrftLbMI+A=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
```

Comment:
TIL `dependencies`! I've always used `propagatedBuildInputs` but according to the doc, this is even better.

Reply:
As proposed by @dotlambda [here](https://github.com/NixOS/nixpkgs/issues/343042#issuecomment-2361261030)